src/components/containers/LectureCard/EventCard.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import styles from './EventCard.module.scss'
+import {LocationChip} from "./chips/LocationChip";
+import {StartDateChip} from "./chips/StartDateChip";
+
+interface EventCardProps {
+    id: string,
+    title: string,
+    subtitle: string,
+
+    color?: string,
+    image?: string,
+
+    startDate?: Date,
+    locationName?: string
+}
+
+export const EventCard: React.FC<EventCardProps> = ({
+    title,
+    subtitle,
+    color,
+    image,
+    startDate,
+    locationName,
+}) => {
+
+    const cardStyle: React.CSSProperties = {
+        backgroundColor: color || "#080736" //space dark
```

Comment:
Extract default color somewhere else

Reply:
yeah, gonna move this to react's Component.defaultProps below